### PR TITLE
#78 add request new partnership form

### PR DESCRIPTION
**Pull request by jeliwa24**
image

### CODE REVIEW

File: layouts/athlete.vue [hunk 1]
```diff
@@ -57,8 +57,10 @@
       </v-toolbar-items>
     </v-toolbar>
 
-    <nuxt />
-
+    <v-content>
+      <nuxt />
+    </v-content>
```
Comment (shadowlion) [inline layouts/athlete.vue]: What's the rationale for changing this?

File: pages/u/athlete/partnership/new.vue [hunk 2]
```diff
@@ -1,15 +1,82 @@
...
+              v-model="message" 
+              :rules="[rules.required]"
+              hint="Please do not put any sensitive or important information here." />
+          </v-flex>
+        </div>
+        <v-btn
+          color="info"
+          @click="onSubmit">Submit</v-btn>
+      </v-form>
+    </v-container>
```
Comment (shadowlion) [inline pages/u/athlete/partnership/new.vue]: One indent over to the left plz (maybe?)

File: pages/u/athlete/partnership/new.vue [hunk 3]
```diff
@@ -1,15 +1,82 @@
...
 </template>
 
 <script>
+import { required, email } from "../../../../helpers/validationRules";
```
Comment (shadowlion) [inline pages/u/athlete/partnership/new.vue]: You can use `@` for root, such that it turns into:
```javscript
import { required, email } from "@/helpers/validationRules";
```

File: layouts/athlete.vue (hunk 1 above)
Comment (jeliwa24) [reply layouts/athlete.vue]: without v-content, the page contents overlap with the header

**Pull request merged**